test/boost/cdc_test.cc

```diff
@@ -1126,6 +1126,13 @@ SEASTAR_THREAD_TEST_CASE(test_change_splitting) {
             }
```

Comment:
Can we convert it to just `BOOST_REQUIRE_EQUAL`, just like in line 1181?

Reply:
I checked and we can't. The order is still arbitrary.